conf/app.routes

```diff
@@ -83,3 +83,6 @@ POST        /:taxYear/annual-allowance/pension-provider-paid-tax
 
 GET         /:taxYear/annual-allowance/pension-provider-paid-tax-amount                     controllers.pensions.annualAllowance.TaxPaidByPensionProviderAmountController.show(taxYear: Int)
 POST        /:taxYear/annual-allowance/pension-provider-paid-tax-amount                     controllers.pensions.annualAllowance.TaxPaidByPensionProviderAmountController.submit(taxYear: Int)
+
+GET         /:taxYear/annual-allowance/pension-scheme-tax-reference                         controllers.pensions.annualAllowance.PensionSchemeTaxReferenceController.show(taxYear: Int, pensionSchemeTaxReference: Option[Int])
```

Comment:
Could you rename the parameter for the GET and POST  as it's not actually a pensionSchemeTaxReference but an index
Either pensionSchemeTaxReferenceIndex of similar (pstrIndex or just index?)
The array is called pensionSchemeTaxReference too which makes the code confusing